pterodactyl_exporter/http_client.py

```diff
@@ -47,9 +47,9 @@ def get_server(list_type="owner"):
     for x in json.loads(servers.read())['data']:
         srv["name"].append(x['attributes']['name'])
         srv["id"].append(x['attributes']['identifier'])
-        srv["max_memory"].append(x['attributes']['limits']['memory'])
-        srv["max_swap"].append(x['attributes']['limits']['swap'])
-        srv["max_disk"].append(x['attributes']['limits']['disk'])
+        srv["max_memory"].append(x['attributes']['limits']['memory'] * MiB_TO_MB)
```

Comment:
Just for clarification.
Shouldn't we then also move the modification of the response value from for example line 70 to the get_metrics function to keep it consistent?

Reply:
heyo
Now I looked into it. 
In my opinion it does make sense to convert the values in this place.
Because only `max_memory`, `max_swap` and `max_disk` are served in MiB from Pterodactyl instead of bytes or MB.
The other values which you append in `get_metrics()` are in bytes so `/1000000` does convert them to MB. 

And in the metrics you provide with your exporter is written that you provide them in MB instead of MiB so we need to convert them in this place. 

I can also convert the values in `get_metrics()` but this will lead into more code and is not necessary in my opinion.

Please tell me what you think :)

Reply:
Now I get what your on to. Indeed, this is a breaking change which Pterodactyl did to their API. What I meant yesterday is that, for me it makes more sense to completely switch over from MB to MiB to go with the values shown on the panel. In the past the values on the panel were in MB, that is why I went with that in the first place.

Reply:
Okay :D
For me I just remember the values in MiB/GiB and not MB/GB.
But in this case we also need to change the name of the metrics and this will result in "data loss" and you also need to update the grafana panel :/ :D 
Not big issue but for those who are using this exporter a long time it is maybe not the best solution to switch to MiB instead of MB.

Reply:
Well, for me it's no big deal changing the metrics names, but you are correct about the data loss. I don't know if this would be such a big problem, because I don't usually review metrics for long time spans. My longest used interval is about 1 month, so for me it would be fine. But I'll leave it to you to decide whether to change it to MiB or keep it as MB. I usually think in MiB/GiB too and it would just be a logical step to switch that.

Reply:
Yes, please do so.

Reply:
heyo
I took the chance and looked a bit more in to python programming and made a tiny rewrite of your code :D
I tried to make some form of validation for the config. I hope this is okay. 
I know python dgaf about types and so on but it was a funny task 😄 

I hope it is okay for you. If not I will just make a PR where I only fix the bugs I mentioned in my ticket ^^

I tried to cover it with some unit tests which I generated with chatgpt xD